ui/src/app/core/collection/collection.service.ts

```diff
@@ -52,4 +54,28 @@ export class CollectionService {
       headers: { 'x-api-key': apiKey },
     });
   }
+
+  getSubjectExampleList(apiKey: string): Observable<SubjectExampleResponseItem[]> {
+    return this.http.get(`${environment.userApiUrl}recognition/faces?size=1000`, {
+      headers: { 'x-api-key': apiKey },
+    }).pipe(
+      map((resp: { faces: SubjectExampleResponseItem[] }) => resp.faces.map(face => face))
```

Comment:
looks like excessive array.map (resp.faces.map(face => face)). Can be replaced with resp.faces

Reply:
fixed